E3Next/Settings/GeneralSettings.cs

```diff
@@ -54,6 +54,7 @@ public int Casting_DefaultSpellGem
         public List<string> Loot_OnlyStackableAlwaysLoot = new List<string>();
         public Int32 Loot_OnlyStackableValueGreaterThanInCopper = 1;
         public Boolean Loot_OnlyStackableEnabled = false;
+        public Int32 Loot_TimeToWaitAfterAssist = 10000;
```

Comment:
lets set this to like 2 seconds default.

Reply:
Can do, just fyi though with 4000ms I was running into some cases of looting starting during fights on a pofire swarm